storage_service/locations/models/package.py

```diff
@@ -3247,18 +3250,17 @@ def _get_compression_details_from_premis_events(premis_events, aip_uuid):
     return compression_event.compression_details
 
 
-def _replicate_package_mdl_inst(package_mdl):
+def _replicate_package_mdl_inst(package):
     """Create a new Django ``Package`` instance that is exactly like
     ``package_mdl`` but which has a new primary key and UUID, and references
     ``package_mdl`` as its ``replicated_package``.
```

Comment:
Should these `package_mdl`s in the docstring be updated?

Reply:
Good point! I changed the naming of this method in 80efe62, should be clearer now.

Reply:
It is! Thank you!